cluster/saltbase/salt/kube-controller-manager/kube-controller-manager.manifest

```diff
@@ -12,9 +12,6 @@
 {% if pillar['allocate_node_cidrs'] is defined -%}
   {% set allocate_node_cidrs = "--allocate-node-cidrs=" + pillar['allocate_node_cidrs'] -%}
 {% endif -%}
-{% if pillar['enable_experimental_api'] is defined -%}
-  {% set enable_experimental = "--enable-experimental=" + pillar['enable_experimental_api'] -%}
-{% endif -%}
 
```

Comment:
```console
$ $ grep enable_experimental[^_] -l -R cluster
cluster/saltbase/salt/kube-controller-manager/kube-controller-manager.manifest
```
Looks like just the one use currently.  And I think apiserver will determine what resources to enable from `runtime-config`, so we won't need the pillar there.